.github/workflows/pr.yaml

```diff
@@ -206,20 +205,19 @@ jobs:
       - name: Built tracee BPF
         run: |
           make -f Makefile.one bpf-core
-      - name: Build tracee-core-btfhub
+      - name: Build tracee container image (slim w/ BTFHUB)
         run: |
-          make -f builder/Makefile.tracee-container build-alpine-tracee-core-btfhub
+          BTFHUB=1 make -f builder/Makefile.tracee-container build-tracee
```

Comment:
@danielpacak could you check if BTFHUB=1 was passed to the docker build correctly AND the tracee:latest had, indeed, a tracee-ebpf binary with embedded tailored BTF files on it ? If that is true, we would be testing CO-RE.

I believe it worked because I told "go test" to use "tracee:latest" and it wouldn't have worked without BTFHUB support.

But it could be that the non core eBPF object was still in /tmp/tracee and then we would be testing the non CO-RE again here. Perhaps `rm -rf /tmp/tracee` in between the 2 tests ? Not sure.

Reply:
@danielpacak, could you take a look at the failing tests for me ? With this new container images, the smoke tests seem flaky to me. They are passing locally:

```
$ go test -v -run "TestTraceeSignatures" ./tests/tracee_test.go -tracee-image-ref "tracee:latest" -tracee-tester-image-ref "aquasec/tracee-tester:latest"  -tracee-signatures "TRC-2,TRC-3"
=== RUN   TestTraceeSignatures
=== RUN   TestTraceeSignatures/tracee:latest/TRC-2
2022/02/04 01:29:42 Starting container id: e478f0c3163a image: testcontainers/ryuk:0.3.3
2022/02/04 01:29:43 Waiting for container id e478f0c3163a image: testcontainers/ryuk:0.3.3
2022/02/04 01:29:43 Container is ready id: e478f0c3163a image: testcontainers/ryuk:0.3.3
2022/02/04 01:29:43 Starting container id: 971112419f47 image: tracee:latest
2022/02/04 01:29:44 Waiting for container id 971112419f47 image: tracee:latest
2022/02/04 01:29:48 Container is ready id: 971112419f47 image: tracee:latest
2022/02/04 01:30:03 Starting container id: 79c7cd6872d1 image: aquasec/tracee-tester:latest
2022/02/04 01:30:03 Container is ready id: 79c7cd6872d1 image: aquasec/tracee-tester:latest
=== RUN   TestTraceeSignatures/tracee:latest/TRC-3
2022/02/04 01:30:24 Starting container id: 6e05b2a5e190 image: tracee:latest
2022/02/04 01:30:25 Waiting for container id 6e05b2a5e190 image: tracee:latest
2022/02/04 01:30:29 Container is ready id: 6e05b2a5e190 image: tracee:latest
2022/02/04 01:30:29 Starting container id: 5f763d8b7e91 image: aquasec/tracee-tester:latest
2022/02/04 01:30:30 Container is ready id: 5f763d8b7e91 image: aquasec/tracee-tester:latest
--- PASS: TestTraceeSignatures (73.66s)
    --- PASS: TestTraceeSignatures/tracee:latest/TRC-2 (47.16s)
    --- PASS: TestTraceeSignatures/tracee:latest/TRC-3 (26.48s)
PASS
ok  	command-line-arguments	73.673s
```

And looks like they're randomly failing in github actions. I tried raising timeout values, not sure if that helped.

Feel free to rebase this change in another PR of yours, with fixes, if you can find any issue.

Reply:
This has been resolved, hopefully, by https://github.com/aquasecurity/tracee/pull/1439 with the change of pulling the image before the initial container needed it.

@danielpacak and I discovered that docker background daemon was causing too many events when downloading image during the container initialization and the tests were flaky because of that (I'll explain in another case).

For now this was mitigated by making docker to pull the image (before it did not happen because we were building the container test image instead of pulling it during first container initialization).

more to come, for now this can be considered "mitigated" (or should be).